packages/css/src/components/table-of-contents/table-of-contents.scss

```
@@ -0,0 +1,72 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@import "../../common/text-rendering";
+
+@mixin reset {
+  box-sizing: border-box;
+  margin-block: 0;
+  padding-inline: 0;
+}
+
+.ams-table-of-contents {
+  display: flex;
+  flex-direction: column;
+  font-family: var(--ams-table-of-contents-font-family, inherit);
+  font-size: var(--ams-table-of-contents-font-size);
+  font-weight: var(--ams-table-of-contents-font-weight);
+  gap: var(--ams-table-of-contents-gap);
+  line-height: var(--ams-table-of-contents-line-height);
+}
+
+.ams-table-of-contents__list {
+  display: flex;
+  flex-direction: column;
+  gap: var(--ams-table-of-contents-list-gap);
+  list-style: none;
+
+  .ams-table-of-contents__list {
+    padding-block-start: var(--ams-table-of-contents-list-gap);
+    padding-inline-start: var(--ams-table-of-contents-list-list-padding-inline-start);
+  }
+
+  @include text-rendering;
+  @include reset;
```

Switching these blocks completes the declarations for the list before styling another element.

```suggestion

  @include text-rendering;
  @include reset;

  .ams-table-of-contents__list {
    padding-block-start: var(--ams-table-of-contents-list-gap);
    padding-inline-start: var(--ams-table-of-contents-list-list-padding-inline-start);
  }
```